server/README.md

```diff
@@ -14,3 +14,27 @@ This directory contains all server code for the Strimzi UI - ie code which is re
 - `serverConfig.ts` - configuration handling module. If an `configPath` envar is provided, the file specified is used for the server's configuration, else, `server.config.json` in the current working directory will be used. If neither exist, a default config defined in this module will be used.
 - `types.ts` - custom types and interfaces used in the server code base.
 - `tsconfig.json` - Typescript config for this codebase.
+- `jest.config.js` - Jest config for this codebase.
+
+## Configuration options
+
+As described in [the configuration approach](../docs/Architecture.md#configuration-and-feature-flagging), the UI server's configuration is provided via a file, which is then watched at runtime for modification. This configuration file is expected to be called `server.config.json` (available in the same directory as the `node` executable is run from), but this can be configured at runtime via environment variable `configPath`, dictating a different path and file name. The file must be either valid JSON or JS. The server also hosts configuration for discovery by the client via the `config` module. The configuration options for the server provided in the previously mentioned configuration file are as follows:
+
+| Configuration                | Required | Default                                                                                                                | Purpose                                                                                                                                                                                |
+| ---------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| authentication.strategy      | No       | `none`                                                                                                                 | What authentication strategy to use to authenticate users. See [the security section](#security) for details of the available options.                                                 |
```

Comment:
Overlap with security PR but this needs `authorization.strategy/configuration` too

Reply:
👍 - Would this come as a part of the security work? Main motivation with the movement of the config table from architecture to here was to slim down the architecture doc/keep the config with the code it acts on

Reply:
Yep that's fine :)